### PR TITLE
🔕 Silent send messages front-end

### LINKED ISSUE #7366: Add option for "🔕 Silent send" to Web

### PR DESCRIPTION
**Pull request by marcoambrosini**
Fix #7366

### CODE REVIEW

**Comment by marcoambrosini** [comment]
removed the TS changes from here for now since it's harder than I thought, so that this can go through

File: src/store/messagesStore.js [hunk 1]
```diff
@@ -855,9 +855,12 @@ const actions = {
 	 * Sends the given temporary message to the server.
 	 *
 	 * @param {object} context default store context;
+	 * @param temporaryMessage.temporaryMessage
 	 * @param {object} temporaryMessage temporary message, must already have been added to messages list.
+	 * @param {object} options post request options.
+	 * @param temporaryMessage.options
 	 */
```
Comment (nickvergessen) [inline src/store/messagesStore.js:862]:
```suggestion
	 * @param {object} context default store context;
	 * @param {object} data Passed in parameters
	 * @param {object} data.temporaryMessage temporary message, must already have been added to messages list.
	 * @param {object} data.options post request options.
	 */
```

File: src/store/messagesStore.js [hunk 2]
```diff
@@ -855,9 +855,12 @@ const actions = {
 	 * Sends the given temporary message to the server.
 	 *
 	 * @param {object} context default store context;
-	 * @param {object} temporaryMessage temporary message, must already have been added to messages list.
+	 * @param {object} context default store context;
```
Comment (nickvergessen) [inline src/store/messagesStore.js:858]: Duplicate @param "context"

File: src/components/NewMessageForm/NewMessageForm.vue [hunk 3]
```diff
@@ -105,17 +105,32 @@
...
-				</Button>
+				<!-- Send buttons -->
+				<template v-else>
+					<Actions :force-menu="true">
+						<!-- Silent send -->
+						<ActionButton :close-after-click="true"
+							icon="icon-upload"
+							:title="t('spreed', 'Send silently')"
+							@click.prevent="handleSubmit({ silent: true })">
+							{{ t('spreed', 'The recipients won\'t be notifyed about this message') }}
```
Comment (nickvergessen) [inline src/components/NewMessageForm/NewMessageForm.vue:116]:
```suggestion
							{{ t('spreed', 'The recipients won\'t be notified about this message') }}
```

Also maybe we put this in a computed and for one to one chats we then use:
```suggestion
							{{ t('spreed', 'The recipient won\'t be notified about this message') }}
```

Other than that should it be "participants?

**Review by nickvergessen — CHANGES_REQUESTED** [review]

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**